tests/console/btrfs_qgroups.pm

```diff
@@ -74,9 +74,9 @@ sub run {
 
     # Check for quota exceeding
     assert_script_run 'btrfs subvolume create e';
-    assert_script_run 'btrfs qgroup limit 50m e .';
-    my $write_chunk = 'dd if=/dev/zero bs=1M count=40 of=e/file';
-    assert_script_run "for c in {1..2}; do $write_chunk; done", fail_message => 'bsc#1019614 overwriting same file should not exceed quota';
+    assert_script_run 'btrfs qgroup limit 200m e .';
+    my $write_chunk = 'dd if=/dev/zero bs=1M count=190 of=e/file';
+    assert_script_run "for c in {1..2}; do $write_chunk; done", fail_message => 'bsc#1113042 overwriting same file should not exceed quota';
```

Comment:
I think that the text should also be updated.

```suggestion
# Overwriting same file should not exceed quota
    assert_script_run "for c in {1..2}; do $write_chunk; done", fail_message => 'bsc#1113042 btrfs is not informed to commit transaction';
```

Reply:
done